internal/common/allow.go

```diff
@@ -0,0 +1,81 @@
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+// the ALLOW_FILE file must conform to the following schema:
+// {
+// 	"default":{
+//		"rhel-84": true,
+//		"rhel-85": true,
+//		"rhel-86": true,
+//		"rhel-90": true,
+//		"centos-8": true,
+//		"centos-9": true,
+// 	},
+// 	"000000":{
```

Comment:
I wonder if accepting glob patterns would make this more convenient:
```
{
  "123456": [
    "fedora-*"
  ]
}
```

Reply:
Yep, and done!